xclim/indices/_agro.py

```diff
@@ -507,6 +508,133 @@ def cool_night_index(
     return cni
 
 
+@declare_units(pr="[precipitation]", evspsblpot="[precipitation]")
+def dryness_index(
+    pr: xarray.DataArray,
+    evspsblpot: xarray.DataArray,
+    lat: xarray.DataArray | str | None = None,
+    wo: str = "200 mm",
+    freq: str = "YS",
+) -> xarray.DataArray:
+    """Dryness Index.
+
+    Mean minimum temperature for September (northern hemisphere) or March (Southern hemisphere).
```

Comment:
This description is for `cool_night_index`,  it doesn't seem related to `dryness_index`, right?

Reply:
Woops, yes. That was a copy-paste. Will address.